packages/components/src/components/select-native/select-native.scss

```diff
@@ -59,9 +59,6 @@
             color: var(--text-general);
             max-width: 100%;
         }
-        &--disabled {
-            color: var(--text-disabled);
-        }
```

Comment:
why are removing this?

Reply:
Currently, other disabled input components do not have grey labels, only this dropdown stands out and makes it inconsistent.

Reply:
I checked, only dropdown labels on mobile were grey out, the rest keep their color on mobile and desktop, so I think we need to keep it consistent for all